doc/api/util.md

```diff
@@ -3707,6 +3710,7 @@ util.isArray({});
 [`'warning'`]: process.md#event-warning
 [`Array.isArray()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
 [`ArrayBuffer.isView()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/isView
+[`Error.isError`]: https://github.com/tc39/proposal-is-error
```

Comment:
Is there a reason to use link to the proposal rather than to MDN?

Reply:
No good reason, thanks